theano/sandbox/gpuarray/dnn.py

```diff
@@ -75,18 +75,15 @@ def _dnn_check_compile():
         try_run=False, output=True)
 
     if not avail:
-        return False, ("Theano cannot compile with cuDNN. "
+        return False, ("cannot compile with CuDNN. "
                        "We got this error:\n" + str(err))
     return True, None
 
 
 def _dnn_check_version():
     v = version()
-    if v < 3007:
-        return False, (
-            "You have an old release of CuDNN (or a release candidate) "
-            "that isn't supported.  Please update to at least v3 final "
-            "version.")
+    if v < 4007:
+        return False, "Version too old. Update to v5."
```

Comment:
Same as above.

As we have multiple dependencies, I think it's important to know the version of what is too old.
Maybe "Your cuDNN version is too old. Update to v5."